modules/33acrossBidAdapter.js

```diff
@@ -25,6 +27,7 @@ function _createBidResponse(response) {
 function _createServerRequest(bidRequest) {
   const ttxRequest = {};
   const params = bidRequest.params;
+  const url = (params.test === 1) ? (TEST_END_POINT) : (END_POINT);
 
```

Comment:
No more setting test endpoint via params. The param `test: 1` will take care of this